app/src/main/java/com/twilio/video/app/ui/room/RoomActivity.java

```diff
@@ -1764,6 +1811,27 @@ public void onVideoTrackDisabled(
         }
     }
 
+    private @DrawableRes int getNetworkQuality(NetworkQualityLevel networkQualityLevel) {
+        int result = R.drawable.network_quality_level_0;
+
+        if (networkQualityLevel == NetworkQualityLevel.NETWORK_QUALITY_LEVEL_UNKNOWN
```

Comment:
In iOS and in the Android Multiparty example we don't show any indicator image for `NETWORK_QUALITY_LEVEL_UNKNOWN`. It can take a few second to get the first score and showing no network is a bit misleading.

Shouldn't be too hard to change. When it's unknown, hide the image, when it's anything else, unhide it and update with the proper image.